spec/features/functions/multiple_identities_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe "Functions with multiple definitions", :db do
```

Comment:
Are the identities the factor that varies? I would see it as the arguments.

Could we clarify this test description somehow?